### ISSUE TITLE
setup: support install from debian package repository

### ISSUE THREAD

**Issue by magandrez** [Azure/azure-cli#13336]
## Describe the bug

**Command Name**
`az acr create`

**Errors:**
```
cannot import name 'BlockBlobService' from 'azure.storage.blob' (/usr/lib/python3/dist-packages/azure/storage/blob/__init__.py)
Traceback (most recent call last):
python3/dist-packages/knack/cli.py, ln 206, in invoke
    cmd_result = self.invocation.execute(args)
cli/core/commands/__init__.py, ln 528, in execute
    self.commands_loader.load_arguments(command)
azure/cli/core/__init__.py, ln 299, in load_arguments
    self.command_table[command].load_arguments()  # this loads the arguments via reflection
...
cli/command_modules/acr/_archive_utils.py, ln 15, in <module>
    from azure.storage.blob import BlockBlobService
ImportError: cannot import name 'BlockBlobService' from 'azure.storage.blob' (/usr/lib/python3/dist-packages/azure/storage/blob/__init__.py)
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az acr create --resource-group {} --name {} --sku {}`

## Environment Summary
```
Linux-5.5.0-2-amd64-x86_64-with-glibc2.29
Python 3.8.3rc1
Shell: zsh

azure-cli 2.0.81

Extensions:
azure-devops 0.18.0

```

**Comment by yonzhan** [comment]
acr

**Comment by yungezz** [comment]
hi @magandrez have you tried on latest Azure CLI version?

**Comment by yugangw-msft** [comment]
This is not ACR issue

**Comment by yugangw-msft** [comment]
@magandrez, to cross check, are you able to run any storage command like "az storage blob list"?

**Comment by magandrez** [comment]
@yungezz Yes indeed. I realised I was running the CLI available in Debian BullsEye at v2.0.81. I then used the sources list from Microsoft (available in the install instructions for Azure CLI) and installed latest version (2.5.1)

@yugangw-msft With latest version installed, yes, I am able to perform such actions (see above).

It would be very beneficial to get the package in Debian package repository. At the moment even sid (testing) is locked at 2.0.81 [1]

The maintainer for the package is available in the Debian Package list page for azure-cli [2]

[1] https://packages.debian.org/search?keywords=azure-cli
[2] https://packages.debian.org/bullseye/azure-cli
